habitat/tasks/nav/nav_task.py

```diff
@@ -458,7 +466,7 @@ def get_original_map(self, episode):
             point_padding = 2 * int(
                 np.ceil(self._map_resolution[0] / MAP_THICKNESS_SCALAR)
             )
-            top_down_map[
+            self._top_down_map[
                 s_x - point_padding : s_x + point_padding + 1,
```

Comment:
Is a `+ 1` needed here?

Reply:
This is part of danielgordon10's original implementation that I just left intact. I do think, however, that I should make the if check outside of the function call to make the logic more clear.